src/resources/app-fr.json

```diff
@@ -70,5 +70,8 @@
   "lastEdit": "Dernière modification",
   "updated": "Mise à jour",
   "actions": "Actions",
-  "takeMeToTheResponseData": "Emmenez-moi aux données de réponse"
+  "takeMeToTheResponseData": "Emmenez-moi aux données de réponse",
```

Comment:
Have you also added these translations to the Google Sheets? If no, pls add :)
https://docs.google.com/spreadsheets/d/1XLI0t3heBJ366I36D8pUbHLz8xWV4Um-l965sa5G6PU/edit#gid=0
https://docs.google.com/spreadsheets/d/1iZqlmNywaGDh3P8XenuklTKqTw35fEqSkNBAhEbbm-0/edit#gid=533595104